### PR TITLE
Passport rewrite

### PR DESCRIPTION
**Pull request by stancl**
Based on #184

### CODE REVIEW

**Comment by stancl** [comment]
@lukinovec Could you make these links not bold and find why they don't seem to work?

CleanShot 2022-07-29 at 8 22 28@2x

See here, the # fragment doesn't seem to do anything on my end. See the names of the headings used.

https://62e4246a0456390008c096f6--stancl-tenancy.netlify.app/docs/v3/integrations/passport

**Comment by lukinovec** [comment]
@stancl, the links are working now (added "{#heading-name}" to the headings). There are some things to improve:

- At the beginning, we urge the user not to use `passport:install` to avoid issues, but in the "Using Passport only in central application" section, we instruct them to set up Passport following the Laravel docs where `passport:install` is used as a part of the installation process. We should mention what to do instead of using `passport:install`, or if the reminder about `passport:install` even applies for that section.
- We should improve the writing and formatting (e.g. some unnecessarily bold text).

**Comment by stancl** [comment]
> We should improve the writing and formatting (e.g. some unnecessarily bold text)

Feel free to do that as you see fit

File: source/docs/v3/integrations/passport.blade.md [hunk 1]
```diff
@@ -6,107 +6,123 @@ section: content
...
-Also change the value of `storage.database.connection` to `null` in the file `config/passport.php` to force Passport to use the default database connection. That way, Passport will work in both central and tenant parts of the application.
+If you want to use an unique Passport keys for each tenant, there are multiple ways you can store and load tenant Passport keys, but the most straightforward way is to store the keys in the `Tenant model` and load them into the passport configuration using the [Tenant Config]({{ $page->link('features/tenant-config') }}) feature.
+
+Once the [Tenant Config]({{ $page->link('features/tenant-config') }}) feature is enabled, simply map your tenant Passport keys into the `boot` method of your `TenancyServiceProvider` as follows:
+    ```php
+    \Stancl\Tenancy\Features\TenantConfig::$storageToConfigMap = [
+        'passport_public_key' => 'passport.public_key',
+        'passport_private_key' => 'passport.private_key',
+    ],
+    ```
```
Comment (darkons) [inline source/docs/v3/integrations/passport.blade.md:128]:
```suggestion
```php
\Stancl\Tenancy\Features\TenantConfig::$storageToConfigMap = [
    'passport_public_key' => 'passport.public_key',
    'passport_private_key' => 'passport.private_key',
];
```

**Comment by stancl** [comment]
The PR looks great now. The tenant-specific keys as described on the page don't seem to work, right? Perhaps we could add a warning below the Note in that section saying that the steps haven't been 100% tested.

**Comment by stancl** [comment]
You can add that warning and I'll merge this 👍🏻

**Comment by lukinovec** [comment]
[Added the warning](https://github.com/stancl/tenancy-docs/pull/185/commits/d6b9d81a4fe9f2f29ab2a2fa2faf273fe05930e8) @stancl

**Comment by stancl** [comment]
Looks great! Thanks for your work on this @lukinovec @darkons

**Pull request merged**